tests/homework6/test_oop_2.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+
+from homework6.oop_2 import *
+
+opp_teacher = Teacher("Daniil", "Shadrin")
+advanced_python_teacher = Teacher("Aleksandr", "Smetanin")
+
+lazy_student = Student("Roman", "Petrov")
+good_student = Student("Lev", "Sokolov")
+
+oop_hw = opp_teacher.create_homework("Learn OOP", 1)
+docs_hw = opp_teacher.create_homework("Read docs", 5)
+
+result_1 = good_student.do_homework(oop_hw, "I have done this hw")
+result_2 = good_student.do_homework(docs_hw, "I have done this hw too")
+result_3 = lazy_student.do_homework(docs_hw, "done")
```

Comment:
It would be more appropriate to initialize the environment for tests using pytest fixtures.